WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3425,10 +3425,11 @@ e_roman_empire = {
 	
 	culture = roman
 	religion = nicene
+	primary = yes
 	
 	dynasty_title_names = no
 
-	dignity = 100
+	dignity = 500
```

Comment:
It's the ROMAN EMPIRE. I don't think one hundred units of Dignity is enough for the _**ROMAN EMPIRE**_, There are actually several titles with higher Dignity!